modules/nw-externalip-about.adoc

```diff
@@ -0,0 +1,184 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-externalip.adoc
+
+[id="nw-externalip-about_{context}"]
+= About external IP addresses
+
+You can allow external traffic into the cluster by defining an *external IP* configuration.
+
+The configuration for external IP address blocks is defined by a Network custom resource (CR) named `cluster`.
+The Network custom resource (CR) is defined by the `Network.config.openshift.io` CustomResourceDefinition.
+
+The object is created during cluster installation by the Cluster Version Operator (CVO).
+
+// If it is a cloud deployment, what happens?
+// Is this configuration ignored, or does it break a cloud deployment?
+[IMPORTANT]
+====
+This feature is only supported in non-cloud deployments.
+For cloud deployments, use the load balancer services for automatic deployment of a cloud load balancer to target the service's endpoints.
+====
+
+Use of external IP in {product-title} is governed by the following parameters:
+
+// What happens to existing services if this is changed to another IP block?
+// What happens if policy rules change an address becomes rejected?
+
+* `spec.externalIP.autoAssignCIDRs` defines an IP address block used by the load balancer when choosing an external IP address for the service. {product-title} supports only a single IP address block for automatic assignment.
+* `spec.externalIP.policy` defines the permissible IP address blocks when manually specifying an IP address. {product-title} does not apply policy to IP address blocks defined by `spec.externalIP.autoAssignCIDRs`.
+
+If routed correctly, external traffic from the configured external IP address pools can reach service endpoints through any TCP or UDP port that the service exposes.
+This can be simpler than having to manage the port space of a limited number of shared IP addresses when manually assigning external IPs to services.
+
+[CAUTION]
+====
+You must ensure that the IP address pool you assign terminates at one or more nodes in your cluster.
+====
+
+// Do we offer this in 4.x?
+// Also, these addresses can be used as virtual IPs (VIPs) when configuring xref../admin_guide/high_availability.adoc#configuring-ip-failover[high availability].
+
+{product-title} supports both the automatic and manual assignment of IP
+addresses, and each address is guaranteed to be assigned to a maximum of one
+service. This ensures that each service can expose its chosen ports regardless
+of the ports exposed by other services.
+
+For manually-configured external IPs, potential port clashes are handled on a first-come, first-served basis. If you request a port, it is only available if it has not yet been assigned for that IP address. For example:
```

Comment:
Not sure what you mean by port collisions. We also have to worry about IP collisions if externalips are manually allocated.